src/components/AuthComponent/SignupSection.js

```diff
@@ -1,11 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import { Box, Button, Grid, TextField, Typography, Link } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import { useDispatch } from "react-redux";
-import { loginAction } from "../../services/authService";
+import { loginAction, loginActionWithEmailPassword } from "../../services/authService";
 
 function SignupSection() {
   const dispatch = useDispatch();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [displayName, setDisplayName] = useState("");
```

Comment:
combine these three in one Object like:
```
const [user, setUser] = useState({});
```
 to change the particular key in setState,
 ```
 setUser({...user, key: value})
 ```

Reply:
Sorry! I am unable to combine ,when i combine then error shows of not defined , can you teach me how to do it.

Reply:
Please take a look at how it done.

https://github.com/codeforcauseorg/codeforcause.org/blob/ff1261f05941b4e5618f9ff7890a76e4ac1f91aa/src/views/pages/CLView/ApplyNowModal.js#L47-L67

The formData is the combined that that contains all the information about the form. See how it is used in textFields